includes/class-wc-integrations.php

```diff
@@ -48,4 +48,18 @@ public function __construct() {
 	public function get_integrations() {
 		return $this->integrations;
 	}
+
+	/**
+	 * Fetches all of the default integrations.
+	 *
+	 * @return array
+	 */
+	private function get_default_integrations() {
+		$default_integrations = array();
+
+		include_once WC_ABSPATH . 'includes/integrations/maxmind-geolocation/class-wc-maxmind-geolocation-integration.php';
```

Comment:
It should be part of the `WC_Autoloader` and load any integration class inside `integrations` directory.

Reply:
Doing it we can just list the default integrations array inside `__construct` like we does on `WC_Payment_Gateways` class.

Reply:
Agreed, thanks!